themes/default/content/registry/packages/aws/installation-configuration.md

```diff
@@ -293,6 +293,17 @@ Make sure that your local environment does not have AWS credentials configured b
 
 To learn more about projecting environment variables in Pulumi ESC, refer to the [relevant Pulumi ESC documentation](/docs/pulumi-cloud/esc/environments/#projecting-environment-variables).
 
+
+### Authenticate with WebIdentity and OpenID Connect (OIDC)
```

Comment:
I recommend moving this section above the Pulumi ESC section since this new section more "generic" OIDC configuration content.

Reply:
For example in your section you have

```
you configure an AWS role to assume and a source for a web identity token, which is an OIDC ID token. The token is used to authenticate with AWS and obtain temporary credentials. The temporary credentials are then used to access AWS resources. This mode of authentication allows you to run Pulumi on a service that supports OIDC like GitHub, GitLab, or Azure DevOps, and access AWS without storing credentials.
```

But we are already referencing the usage of OIDC in the ESC section which is above it.

Reply:
agreed, moved it